sopel/module.py

```diff
@@ -674,3 +675,22 @@ def __call__(self, func):
         }
         func.example.append(record)
         return func
+
+
+def output_prefix(prefix):
+    """Decorate a function to add a prefix on its output.
+
+    :param str prefix: the prefix to add (must include trailing whitespace if
+                       desired; Sopel does not assume it should add anything)
+
+    Prefix will be added to text sent through:
+
+      * :meth:`sopel.bot.SopelWrapper.say<bot.say>`
```

Comment:
I think it's over-indented here. Not sure restructuredtext likes it.

Reply:
From a test `make html` run in `/docs/`:

![image](https://user-images.githubusercontent.com/164140/66269816-c5607880-e812-11e9-9469-cabc83384701.png)

Looks OK, but removing the leading spaces does change the output, so I'll take those out in a future PR update. I also need to fix the `:meth:` references; they're broken.